### PR TITLE
Spring cleaning!

### PR DESCRIPTION
**Pull request by EVDW**
😘

### CODE REVIEW

File: .env.dist [hunk 1]
```diff
@@ -9,9 +9,6 @@ GOOGLE_CLIENT_SECRET=
 ### yelp/api ###
 YELP_API_KEY=
 
-###> symfony/framework-bundle ###
+### symfony/framework-bundle ###
 APP_ENV=dev
 APP_SECRET=fafa69ad10e0a0e5ee2b6b9ea33afd24
-#TRUSTED_PROXIES=127.0.0.0/8,10.0.0.0/8,172.16.0.0/12,192.168.0.0/16
-#TRUSTED_HOSTS='^localhost|example\.com$'
-###< symfony/framework-bundle ###
```
Comment (LoicBoursin) [inline .env.dist:17]: You should leave this line as it is used by Symfony to know which lines to remove if framework-bundle is uninstalled. Also for the > removed before symfony/framework-bundle and in other files, this is used as a marker by symfony ;) not a fancy thing :D

File: .gitignore [hunk 2]
```diff
@@ -1,11 +1,8 @@
-
-###> symfony/framework-bundle ###
+### symfony/framework-bundle ###
```
Comment (LoicBoursin) [inline .gitignore:1]: `###> `is used by Symfony

File: src/Symfony/EventListener/RequestListener.php [hunk 3]
```diff
@@ -11,11 +11,8 @@ public function onKernelRequest(RequestEvent $event)
         $request = $event->getRequest();
 
         if ('json' === $request->getContentType()) {
-            $request->request->replace(
-                null !== ($data = \json_decode($request->getContent(), true))
-                ? $data
-                : []
-            );
+            $data = \json_decode($request->getContent(), true);
```
Comment (LoicBoursin) [inline src/Symfony/EventListener/RequestListener.php:14]:
```suggestion
            $data = json_decode($request->getContent(), true);
```

File: src/Symfony/Filter/Restaurant.php [hunk 4]
```diff
@@ -35,40 +35,29 @@ class Restaurant implements YelpFilter
...
+    public array $prices = [];
+    public array $categories = [];
 
     public function __construct(Request $request)
     {
-        $this->location = $request->query->get('location', null);
-
-        if (!$this->location) {
+        if (!$location = $request->query->get('location', null)) {
             throw new \Exception('Location are a mandatory parameter.');
```
Comment (LoicBoursin) [inline src/Symfony/Filter/Restaurant.php:49]:
```suggestion
            throw new \Exception('Location is a mandatory parameter.');
```

File: src/Yelp/Client.php [hunk 5]
```diff
@@ -18,8 +17,7 @@ public function __construct(string $apikey)
     public function request(string $method, string $endpoint, string $filters = ''): ResponseInterface
     {
         $client = new HttpClient();
-
-        $uri = \sprintf(
+        $uri    = \sprintf(
```
Comment (LoicBoursin) [inline src/Yelp/Client.php:20]:
```suggestion
        $uri    = sprintf(
```

**Review by LoicBoursin — APPROVED** [review]

File: src/Yelp/Client.php (hunk 5 above)
Comment (EVDW) [reply src/Yelp/Client.php:20]: Same

**Review by LoicBoursin — APPROVED** [review]

**Pull request merged**